flutter/lib/src/sentry_flutter.dart

```diff
@@ -163,8 +164,16 @@ mixin SentryFlutter {
     // Will catch any errors that may occur in the Flutter framework itself.
     integrations.add(FlutterErrorIntegration());
 
-    // This tracks Flutter application events, such as lifecycle events.
-    integrations.add(WidgetsBindingIntegration());
+    if (MultiViewHelper.isMultiViewEnabled()) {
+      // ignore: invalid_use_of_internal_member
+      options.logger(
+        SentryLevel.debug,
+        '`WidgetsBindingIntegration` is not available in multi-view applications.',
+      );
+    } else {
+      // This tracks Flutter application events, such as lifecycle events.
+      integrations.add(WidgetsBindingIntegration());
```

Comment:
or we move the MultiViewHelper.isMultiViewEnabled check inside of the integration, the logger will be available there

Reply:
I left the logger code, because you told me that you make a new issue, for making the logger available earlier, so it will work when this issue is solved.

But I like the approach of moving it inside of the integration.